ptrail/regularization/goemetric_chull_related.py

```diff
@@ -0,0 +1,279 @@
+"""
+    | Author: Abhishek Gujjar
+"""
+# import rsdepth
+import numpy as np
+import pandas as pd
+from shapely.geometry import Polygon
+from scipy.spatial import distance
+from typing import List, Union
+
+
+def traj_convexhull_wrapper(pts: np.ndarray) -> Union[np.ndarray, float]:
+    """
+    Wraps the calculation of the convex hull for a set of points and handles exceptions.
+
```

Comment:
Follow the documentation structure as it is being used for other functions in the library, please check it out.